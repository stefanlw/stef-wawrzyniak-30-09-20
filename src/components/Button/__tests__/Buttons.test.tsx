import React from 'react';
import renderer from 'react-test-renderer';
import { Button } from '../';

describe('Button', () => {
  it('should match snapshot', () => {
    const component = renderer
      .create(<Button onPress={() => {}}>Some Text</Button>)
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
