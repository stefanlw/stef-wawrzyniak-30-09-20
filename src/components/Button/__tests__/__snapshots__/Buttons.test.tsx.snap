// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should match snapshot 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={true}
  nativeID="animatedComponent"
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "borderRadius": 6,
          "justifyContent": "center",
          "padding": 12,
          "paddingHorizontal": 16,
        },
        Object {
          "backgroundColor": "#4b3cbb",
        },
      ]
    }
  >
    <Text
      style={
        Object {
          "color": "#ffffff",
          "fontSize": 14,
          "fontWeight": "600",
        }
      }
    >
      Some Text
    </Text>
  </View>
</View>
`;
